doc/examples.rst

```diff
@@ -152,3 +152,123 @@ Instantiated ``init.lua`` content:
 
     require("fiber").sleep(1)
 
+Packing environments
+----------------------------------
+
+For example, we want to pack a single application. If there is the next environment structure::
+      single_environment/
+      ├── tarantool.yaml
+      └── init.lua
+
+The contents of ``./tarantool.yaml`` is supposed to be:
+
+.. code-block:: yaml
+
+    tt:
+        app:
+
+For packing it into tarball, call:
+
+.. code-block:: bash
+
+   $ tt pack tgz --without-binaries
+      • Apps to pack: single_environment
+      • Generating new tarantool.yaml for the new package
+      • The passed bundle packed into temporary directory: /var/folders/T/tt_pack1901831662
+      • Creating tarball.
+      • Bundle is packed successfully to /Users/dev/tt_demo/single_environment/single_environment_0.1.0.0.tar.gz.
+
+The result directory structure::
+
+      unpacked_dir/
+      ├── tarantool.yaml
+      ├── single_environment
+      │   └── init.lua
+      ├── env
+      │   ├── bin
+      │   └── modules
+      ├── instances_enabled
+      │   └── single_environment -> ../single_environment
+      └── var
+          ├── lib
+          ├── log
+          └── run
+
+Example of packing a multi-app environment::
+
+     bundle/
+     ├── tarantool.yaml
+     ├── env
+     │   ├── bin
+     │   │   ├── tt
+     │   │   └── tarantool
+     │   └── modules
+     ├── myapp
+     │   ├── Dockerfile.build.cartridge
+     │   ├── Dockerfile.cartridge
+     │   ├── README.md
+     │   ├── app
+     │   ├── bin
+     │   ├── deps.sh
+     │   ├── failover.yml
+     │   ├── init.lua
+     │   ├── instances.yml
+     │   ├── myapp-scm-1.rockspec
+     │   ├── pack-cache-config.yml
+     │   ├── package-deps.txt
+     │   ├── replicasets.yml
+     │   ├── stateboard.init.lua
+     │   ├── systemd-unit-params.yml
+     │   ├── tarantool.yaml
+     │   ├── test
+     │   └── tmp
+     ├── myapp2
+     │   ├── app.lua
+     │   ├── data
+     │   ├── etc
+     │   ├── myapp2
+     │   ├── queue
+     │   ├── queue1.lua
+     │   └── queue2.lua
+     ├── myapp3.lua
+     ├── app4.lua
+     ├── instances_enabled
+     │   ├── app1 -> ../myapp
+     │   ├── app2 -> ../myapp2
+     │   ├── app3.lua -> ../myapp3.lua
+     │   ├── app4.lua -> /Users/dev/tt_demo/bundle1/app4.lua
+     │   └── app5.lua -> ../myapp3.lua
+     └── var
+         ├── lib
+         ├── log
+         ├── pid
+         └── run
+
+``./tarantool.yaml`` content:
+
+.. code-block:: yaml
+
+    tt:
+        app:
+
+Pay attention, that all full links from instances enabled will be resolved, all sources will be copied
```

Comment:
Pls, use "absolute symlink/symbolic link" instead of "full link".
And I'd like to re-phrase this to something like "Absolute symlinks target files/directories will be copied to the package. All relative symlinks in inst_enabled and in apps directories will be converted relatively to the package root directory."

Reply:
Fixed.